src/reason-parser/reason_template.ml

```diff
@@ -0,0 +1,97 @@
+(**
```

Comment:
Looks like this comment is a duplicate of the one in the interface, I understand why you might want to have it here though, this can probably be not a doc comment

Reply:
This diff is actually tiny, but on top of a rebase of two other PRs unmerged. The one you are discussing is actually [here](https://github.com/facebook/reason/pull/2599). Would you mind commenting on the specific PR if you have feedback?

I will likely be aggressively rebasing and force pushing to all of these PRs to keep them clean and comprehensible, where each PR's *top* commit is the real change for that PR with the underlying commits on the PR being other PRs.